lighthouse-core/gather/fetcher.js

```diff
@@ -41,22 +45,24 @@ class Fetcher {
    * So instead we have one global `Fetch.enable` / `Fetch.requestPaused` pair, and allow specific
    * urls to be intercepted via `fetcher._setOnRequestPausedHandler`.
    */
-  async enableRequestInterception() {
+  async enable() {
     if (this._enabled) return;
 
     this._enabled = true;
+    await this.driver.sendCommand('Network.enable');
```

Comment:
can `Network.enable`/`disable` move into `_fetchResourceOverProtocol`? It'll make it easier to lift out `_fetchResourceIframe` when the time comes if the two aren't interleaved.